api/agent/agent.go

```diff
@@ -152,14 +152,23 @@ func New(ds models.Datastore, ls models.LogStore, mq models.MessageQueue) Agent
 		hot:         make(map[string]chan slot),
 		resources:   NewResourceTracker(),
 		shutdown:    make(chan struct{}),
+		stats:       NewStats(),
 		promHandler: promhttp.Handler(),
 	}
 
+	// TODO: move to config
+	statsPollInterval := time.Duration(1) * time.Second
```

Comment:
AFAIK Prometheus polls every 5 seconds, so maybe good to use similar value here?

Reply:
I'm going to close this PR and probably hook this to the new slot mgmt code in https://github.com/fnproject/fn/pull/562